pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+authors = [
+    {name = "Tom Hu", email = "thomas.hu@sentry.io"},
+    {name = "Codecov Support", email = "support@codecov.io"},
+]
+description = "Codecov Command Line Interface"
+requires-python = ">= 3.9"
+dependencies = [
+    "click==8.*",
+    "httpx==0.27.*",
+    "ijson==3.*",
+    "pyyaml==6.*",
+    "regex",
+    "responses==0.21.*",
+    "sentry-sdk>=2.20.0",
+    "test-results-parser==0.5.*",
+    "tree-sitter==0.20.*",
+    "wrapt>=1.17.2",
+]
+license = {file = "LICENSE"}
```

Comment:
Now that PEP 639 has finally landed in full, it's best to use separate `license` SPDX and `license-files`: https://packaging.python.org/en/latest/specifications/pyproject-toml/#license.

`setuptools` hasn't yet implemented it, however:
* https://github.com/pypa/setuptools/issues/4629
* https://github.com/pypa/setuptools/issues/3596
* https://github.com/pypa/setuptools/issues/4759
* https://github.com/pypa/setuptools/pull/4706
* https://github.com/pypa/setuptools/pull/4728

So this is something to do in the future. For now, it would be a good idea to add a `TODO` code comment, so this action item is not lost.

Reply:
`TODO` will probably still get lost, but this [issue](https://github.com/codecov/codecov-cli/issues/605) will make it out